druid/src/core.rs

```diff
@@ -411,6 +418,14 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
             return;
         }
 
+        if !self.is_initialized() {
+            debug_panic!(
```

Comment:
Although I think `debug_panic!` is an excellent addition, I wonder if this shouldn't actually be a `debug_assert!`?

Reply:
`debug_assert!` does nothing in release, while `debug_panic!` at least logs an error. Thus, I would prefer `debug_panic`.

Reply:
makes sense!